### ISSUE TITLE
Sync with Veteran Facing Forms & Authenticated Experience teams on findings across VFF and My VA Form Submission Research and Timely & Truthful Submission research

### ISSUE THREAD

**Issue by mengA6** [department-of-veterans-affairs/va.gov-team#87563]
## Issue Description

Align across our studies and determine next steps, [per Slack](https://dsva.slack.com/archives/C053UDWMH7U/p1719955526577909?thread_ts=1719955235.904579&cid=C053UDWMH7U)

1. VFF team would like an opportunity to observe our testing; I asked Jeana to pass along any outstanding questions they had from their research in case we have time to address in ours.
2. VFF team will invite us to their internal readout of research, expected around the end of July
3. We'll do the same with our research when we're ready
4. Once research is complete for both, the two teams will workshop to see if we can come to a consistent solution based on research and needs for different kinds of forms
5. We'll open the collaboration up to include MyVA, CAIA, and CST

## Reference

1. [2024-05 VFF and My VA Form Submission Research](https://github.com/department-of-veterans-affairs/va.gov-research-repository/issues/554) > [findings](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/Product/2024-05%20VFF%20and%20My%20VA%20Form%20Submission%20Research/Research/Research%20Findings%20for%202024-05%20VFF%20and%20My%20VA%20Form%20Submission%20Research%20Study.md)
2. [our submission status research](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/disability/526ez/research/2024-07-Submission-Status)
3. [Slack thread asking VFF & My VA for their research findings](https://dsva.slack.com/archives/C06R2BZKT62/p1721853422167149)

---
## Tasks
- [x] Review [VFF & My VA research findings](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/Product/2024-05%20VFF%20and%20My%20VA%20Form%20Submission%20Research/Research/Research%20Findings%20for%202024-05%20VFF%20and%20My%20VA%20Form%20Submission%20Research%20Study.md)
- [x] @mengA6 review saved slack threads
- [x] (after our internal research readout where we invite VFF & My VA / Authenticated Experience) Set up meeting & collaborate on agenda - meeting scheduled: `Research Readout > 2024-07-526-submission-status-evaluative-research`
- [x] Conduct meeting
- [x] Note takeaways and next steps - could involve re-ordering our epics, determining when we converge with My VA / CST - meeting scheduled: `Submission Status > cross-team alignment`

## Acceptance Criteria
- [x] Our team, VFF, and MyVA familiar with each other's research findings
- [x] Our team, VFF, MyVA, and OCTO aligned on next steps for each of us

**Comment by pacerwow** [comment]
Let's split this out into three tickets:
1. Sync w VFF about our research findings
2. Incorporate Timely & Truthful Submission research findings into redesign
3. Initiate CAIA feedback once we have redesign task complete

**Comment by mengA6** [comment]
Okay, this ticket is now "Sync with Veteran Facing Forms & Authenticated Experience teams on findings across VFF and My VA Form Submission Research and Timely & Truthful Submission research"

Also created:

- [Incorporate Timely & Truthful Submission research findings into redesign#88519](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/88519)
- [Create CAIA Intake request for Timely & Truthful Submission design#88520](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/88520)

**Comment by mengA6** [comment]
I set up the meetings and have some questions to Shannon on the invite lists [per slack](https://dsva.slack.com/archives/C053UDWMH7U/p1722027634869769?thread_ts=1719955235.904579&cid=C053UDWMH7U)

**Comment by mengA6** [comment]
Communicated next steps [in slack](https://dsva.slack.com/archives/C06R2BZKT62/p1723150183758839)
